src/ODataHttpClient/Models/IResponse.cs

```diff
@@ -0,0 +1,9 @@
+using System.Net.Http.Headers;
+
+namespace ODataHttpClient.Models
+{
+    public interface IResponse
+    {
+        HttpRequestHeaders Headers { get; }
```

Comment:
I think that HttpResponseHeaders is better for Response#Headers 

https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpresponsemessage?view=netstandard-2.0

Reply:
Would you tell me about necessity for Response Interface?
It looks like a useless abstraction for me...

Reply:
@iwate ok, interface removed.